### ISSUE TITLE
[BUG] SteamVR 2.1.10: vrstartup.sh launches in LDLP scout, headset display won't turn on (should use sniper SLR)

### ISSUE THREAD

**Issue by Owen-Benner** [ValveSoftware/SteamVR-for-Linux#661]
**Describe the bug**
Launching SteamVR from Steam gives the following warning:
`vrstartup.sh[11680]: WARNING: launching in legacy LDLP scout, please use sniper SLR`
In `Properties` -> `Compatibility`, the box is unchecked for `Force the use of a specific Steam Play compatibility tool`. If I check the box, `Steam Linux Runtime 2.0 (soldier)` and `Steam Linux Runtime 3.0 (sniper)` are not available in the drop-down menu, only `Steam Linux Runtime 1.0 (scout)` and versions of Proton.

The headset display will not turn on, and a pop-up error appears:
```
Failed to connect to window manager
SteamVR failed to connect to the window manager. You may need to restart your session or SteamVR.
Restart SteamVR
(497)
```
Restarting SteamVR does not resolve the issues. I am running Gnome on Xorg, so my window manager should not be the problem.

Launching `vrstartup.sh` manually with sniper SLR resolves the pop-up error and the headset display works:
`exec ~/.local/share/Steam/steamapps/common/SteamLinuxRuntime_sniper/run ~/.local/share/Steam/steamapps/common/SteamVR/bin/vrstartup.sh`
I was then able to play Minecraft Java with the Vivecraft mod in VR. However, SteamVR Room Setup will not launch, as it expects some files to be in different directories. I will file a separate report on that later.

**To Reproduce**
1. On Pop!_OS 22.04, install Steam for Linux using the .deb from Valve's website.
2. Run Steam through terminal with `steam`.
3. Disable Steam Cloud for SteamVR (this ensures default settings).
4. In `Properties` -> `Compatibility`, uncheck `Force the use of a specific Steam Play compatibility tool`.
5. Install SteamVR.
6. Connect headset.
7. Launch SteamVR.

**Expected behavior**
SteamVR launches under SLR 3.0 (sniper) by default and SLR 3.0 (sniper) is available in drop-down. Headset display works.

**System Information:**
- Distribution: Pop!_OS 22.04 LTS x86_64
- SteamVR version: 2.1.10
- Steam client version: Steam Client Update - December 11th
- Opted into Steam client beta?: No
- Graphics driver version:  Mesa 23.2.1-1pop0\~1699535617\~22.04\~6cd9a72 (LLVM 15.0.7)
- Gist for SteamVR System Information: https://gist.github.com/Owen-Benner/e3aa807025802c91fad84fe795fedef5

**Screenshots**
![image](https://github.com/ValveSoftware/SteamVR-for-Linux/assets/47570247/2121b68f-5203-4a2e-9b0c-c781e2dd27f0)

**Note:** Commenters who are also experiencing this issue are encouraged to include the "System Information" section in their replies.

**Comment by Owen-Benner** [comment]
Posted on accident, still editing.

**Comment by Owen-Benner** [comment]
Edited to specify my window manager is Xorg.

**Comment by Owen-Benner** [comment]
Update: Today it seems like SteamVR wants to run under Proton by default instead. I'm not sure why, but I'm selecting to force LDLP scout for now (so my Linux executables stop getting deleted by Steam) and run SteamVR under SLR 3.0 (sniper) through the terminal.

**Comment by kisak-valve** [comment]
Hello @Owen-Benner, you've described https://github.com/ValveSoftware/steam-for-linux/issues/9875, please give https://github.com/ValveSoftware/steam-for-linux/issues/9875#issuecomment-1853037408 a read.

**Comment by Owen-Benner** [comment]
After moving `~/.steam` to allow Steam to regenerate it (as suggested [here](https://github.com/ValveSoftware/SteamVR-for-Linux/issues/662#issuecomment-1863618377)), and upgrading to the Steam Client Beta to resolve [steam-for-linux#9875](https://github.com/ValveSoftware/steam-for-linux/issues/9875), I tried running SteamVR through Steam again. I still got the warning about launching in LDLP scout, but not `Failed to connect to window manager`, and the headset display worked. I downgraded from Steam Client Beta, and again I got the warning but no pop-up and the headset still worked. I'm not sure what, if any, relation there is between the version of Steam Linux Runtime used and `Failed to connect to window manager`. I will go ahead and close this now.
